apis/kueue/v1beta1/workload_types.go

```diff
@@ -514,8 +514,13 @@ const (
 
 	// WorkloadEvictedByDeactivation indicates that the workload was evicted
 	// because spec.active is set to false.
+	// Deprecated: Please use Deactivated instead.
```

Comment:
```suggestion
	// Deprecated: The reason is not set any longer, it is only kept temporarily to ensure
	// pre-existing deactivated workloads remain deactivated after upgrade from version
	// prior to 0.10. The old condition can be removed in 0.11.
```
WDYT @tenzen-y @PBundyra ?

Reply:
SGTM